bftengine/src/bftengine/ClientsManager.cpp

```diff
@@ -164,20 +178,22 @@ std::unique_ptr<ClientReplyMsg> ClientsManager::allocateNewReplyMsgAndWriteToSto
 
   c.repliesInfo.insert_or_assign(requestSeqNum, getMonotonicTime());
   LOG_DEBUG(CL_MNGR, KVLOG(clientId, requestSeqNum));
-  auto r = std::make_unique<ClientReplyMsg>(myId_, requestSeqNum, reply, replyLength);
+  auto r = std::make_unique<ClientReplyMsg>(myId_, requestSeqNum, reply, replyLength - rsiLength);
 
-  uint32_t numOfPages = r->size() / sizeOfReservedPage();
+  // At this point, the rsi data is not part of the reply
+  uint32_t commonMsgSize = r->size();
```

Comment:
commonMsgSize -> wholeMsgSize? -> fullMsgSize?

Reply:
The common message size the part of the message that has to be identical between all replicas

Reply:
Add a comment explaining that, please.